### PR TITLE
Clarify ContentEncryption as optional

### PR DESCRIPTION
**Pull request by JeromeMartinez**
Not all players support ContentEncryption, let's avoid that players not supporting ContentEncryption are considered as non conforming.

### CODE REVIEW

File: ebml_matroska.xml [hunk 1]
```diff
@@ -1215,7 +1215,8 @@ the bytes that were removed from the beginning of each frames of the track.</doc
   </element>
   <element name="ContentEncryption" path="\Segment\Tracks\TrackEntry\ContentEncodings\ContentEncoding\ContentEncryption" id="0x5035" type="master" maxOccurs="1">
     <documentation lang="en" purpose="definition">Settings describing the encryption used.
-This Element **MUST** be present if the value of `ContentEncodingType` is 1 (encryption) and **MUST** be ignored otherwise.</documentation>
+This Element **MUST** be present if the value of `ContentEncodingType` is 1 (encryption) and **MUST** be ignored otherwise.
+Decoding implementations **MAY** support encryption.</documentation>
```
Comment (robUx4) [inline ebml_matroska.xml:1219]: Same remark, this should be `Matroska Readers`.

**Comment by robUx4** [comment]
While it's debatable whether compression should be handled most of the time. Encryption is definitely optional, as 99% of existing players show.

File: ebml_matroska.xml (hunk 1 above)
Comment (JeromeMartinez) [reply ebml_matroska.xml:1219]: I changed to `Matroska Player` as a `Matroska Reader` does not need to handle that (transporting the opaque packets, splitting them, etc, does not require the support of this feature).

**Review by robUx4 — APPROVED** [review]
LGTM

**Review by retokromer — APPROVED** [review]

**Pull request merged**